xml/admin_gui_dashboard.xml

```diff
@@ -953,9 +954,108 @@
+    <title>Adding OSDs</title>
+    <para>
+     To add new OSDs, follow these steps:
+    </para>
+    <procedure>
+     <step>
+      <para>
+       Verify that there are storage devices <emphasis>available</emphasis> on
+       cluster nodes (to see when a device is <emphasis>available</emphasis>,
+       see <xref linkend="deploy-cephadm-day2-service-osd"/>). Then click the
+       down arrow in the top left of the table heading and select
+       <guimenu>Create</guimenu>. This opens the <guimenu>Create OSDs</guimenu>
+       window.
+      </para>
+      <figure>
+       <title>Create OSDs</title>
+       <mediaobject>
+        <imageobject role="fo">
+         <imagedata fileref="dash_osd_add.png" width="75%"/>
+        </imageobject>
+        <imageobject role="html">
+         <imagedata fileref="dash_osd_add.png" width="75%"/>
+        </imageobject>
+       </mediaobject>
+      </figure>
+     </step>
+     <step>
+      <para>
+       Click <guimenu>Add</guimenu> to add primary storage devices for OSDs,
+       and specify filtering criteria in the top right of the <guimenu>Primary
+       devices</guimenu> table, for example <guimenu>Type</guimenu>
+       <guimenu>hdd</guimenu>. Confirm with <guimenu>Add</guimenu>.
+      </para>
+      <figure>
+       <title>Adding Primary Devices</title>
+       <mediaobject>
+        <imageobject role="fo">
+         <imagedata fileref="dash_osd_add_primary.png" width="100%"/>
+        </imageobject>
+        <imageobject role="html">
+         <imagedata fileref="dash_osd_add_primary.png" width="100%"/>
+        </imageobject>
+       </mediaobject>
+      </figure>
+     </step>
+     <step>
+      <para>
+       Back in the <guimenu>Create OSDs</guimenu> window, optionally add shared
+       WAL and BD devices, or enable device encryption.
+      </para>
+      <figure>
+       <title>Create OSDs with Primary Devices Added</title>
+       <mediaobject>
+        <imageobject role="fo">
+         <imagedata fileref="dash_osd_add_primary_created.png" width="75%"/>
+        </imageobject>
+        <imageobject role="html">
+         <imagedata fileref="dash_osd_add_primary_created.png" width="75%"/>
+        </imageobject>
+       </mediaobject>
+      </figure>
+     </step>
+     <step>
+      <para>
+       Click <guimenu>Preview</guimenu> to view the preview of &drvgrps;
+       specification for the devices you previously added. Confirm with
+       <guimenu>Create</guimenu>.
+      </para>
+      <figure>
+       <title></title>
+       <mediaobject>
+        <imageobject role="fo">
+         <imagedata fileref="dash_osd_add_primary_preview.png" width="75%"/>
+        </imageobject>
+        <imageobject role="html">
+         <imagedata fileref="dash_osd_add_primary_preview.png" width="75%"/>
+        </imageobject>
+       </mediaobject>
+      </figure>
+     </step>
+     <step>
+      <para>
+       New devices will be added to the list of OSDs.
```

Comment:
It probably makes sense to add a note here that there currently is no progress indication in the dashboard that visualizes the OSD creation process and it will take some time for the OSDs to be created. The OSDs will appear in the list once they have been deployed, but one could check the logs (via the Logs page) to see status update messages from the orchestrator.